### PR TITLE
Update college-montmorency.csl

### PR DESCRIPTION
**Pull request by danycoolqc**
I rewrote the style from scratch. Code should be much more cleaner.

### CODE REVIEW

File: college-montmorency.csl [hunk 1]
```diff
@@ -13,15 +12,31 @@
...
+      <term name="month-06" gender="masculine">juin</term>
+      <term name="month-07" gender="masculine">juillet</term>
+      <term name="month-08" gender="masculine">août</term>
+      <term name="month-09" gender="masculine">septembre</term>
+      <term name="month-10" gender="masculine">octobre</term>
+      <term name="month-11" gender="masculine">novembre</term>
+      <term name="month-12" gender="masculine">décembre</term>
+      <term name="ordinal">&amp;lt;sup&amp;gt;e&amp;lt;/sup&amp;gt;</term>
+      <term name="ordinal-01" gender-form="feminine" match="whole-number">&amp;lt;sup&amp;gt;ère&amp;lt;/sup&amp;gt;</term>
+      <term name="ordinal-01" gender-form="masculine" match="whole-number">&amp;lt;sup&amp;gt;er&amp;lt;/sup&amp;gt;</term>
```
Comment (rmzelle) [inline college-montmorency.csl]: @danycoolqc, just curious: what software did you test these superscripted ordinal suffixes with? (@adam3smith, do you think this is robust enough and acceptable?)

Comment (danycoolqc) [inline college-montmorency.csl]: Hi,

I only tested with Zotero and Word 2016 (both Mac and PC). I took the idea there : https://forums.zotero.org/discussion/18074/edition-with-superscript

**Comment by rmzelle** [comment]
@adam3smith, can you merge this, assuming you're okay with the HTML-superscripting of ordinals?

**Comment by danycoolqc** [comment]
Hi guys! Do you think it is reasonable to hope it can be merged before August 21st? Our new semester is beginning on August 22nd and it would be great if the new version of the style was in the style repository.

**Comment by rmzelle** [comment]
Sure. Sorry for the long wait.

(@adam3smith, still interested in hearing your thoughts on the superscripted ordinals)

**Pull request merged**